src/Qowaiv/Financial/InternationalBankAccountNumber.Patterns.cs

```diff
@@ -19,16 +19,17 @@ namespace Qowaiv.Financial;
         Bban(Country.AL, "8n,16c"),
         Bban(Country.AO, "21n"),
         Bban(Country.AT, "16n"),
-        Bban(Country.AZ, "4c,20n"),
+        Bban(Country.AZ, "4a,20c"),
```

Comment:
Why so many changes in patterns, the description of this PR only mentions four countries?

Reply:
I did not want to mention this kind of (relatively) small changes. From numeric to alphanumeric obviously ment that previously IBAN's could have been rejected incorrectly (the other way around is not an issue). But you've got a point, these are also changes.